File: programme/degree-to-radian/degree-to-radian.go
// Write a programme to convert degree to radian
package main

import (
	"fmt"
	"math"
)

func main() {
	var degree float64
	fmt.Print("Enter a degree : ")
	fmt.Scan(&degree)
	fmt.Println("Radian of", degree, "degree is", math.Round((degree*(math.Pi/180))*100000)/100000)
}

File: programme/degree-to-radian/degree-to-radian.c
// Write a programme to convert degree to radian
#include <stdio.h>

int main()
{
  float degree;
  printf("Degree : ");
  scanf("%f", &degree);

  printf("Radian : %f\n", 3.141592653589793 * degree / 180);

  return 0;
}
